src/components/Calculator.js

```diff
@@ -21,29 +33,29 @@ class Calculator extends Component {
           </div>
 
           <div className="operators">
-            <button type="button" className="press-btn">AC</button>
-            <button type="button" className="press-btn">+/-</button>
-            <button type="button" className="press-btn">%</button>
-            <button type="button" className="press-btn amber">÷</button>
-
-            <button type="button" className="press-btn-1">7</button>
-            <button type="button" className="press-btn-1">8</button>
-            <button type="button" className="press-btn-1">9</button>
-            <button type="button" className="press-btn amber">X</button>
-
-            <button type="button" className="press-btn">4</button>
-            <button type="button" className="press-btn">5</button>
-            <button type="button" className="press-btn">6</button>
-            <button type="button" className="press-btn amber">-</button>
-
-            <button type="button" className="press-btn">1</button>
-            <button type="button" className="press-btn">2</button>
-            <button type="button" className="press-btn">3</button>
-            <button type="button" className="press-btn amber">+</button>
-
-            <button type="button" className="press-btn-2">0</button>
-            <button type="button" className="press-btn">.</button>
-            <button type="button" className="press-btn amber">=</button>
+            <button onClick={this.clickEvent} name="AC" type="button" className="press-btn">AC</button>
+            <button onClick={this.clickEvent} name="" type="button" className="press-btn">+/-</button>
```

Comment:
- [ ] Good job so far, but this button doesn’t work, you forgot to put the name. Please kindly fix that 🙃 🙃